### ISSUE TITLE
PHP-JSON7 missing in umpleonline docker image.

### ISSUE THREAD

**Issue by Gordonei** [umple/umple#1451]
## Specify a Brief Description

Brief, one or two sentences explaining the problem. 

When running the `umple/umpleonline` docker image, I get the following error in the container logs:
```
PHP message: PHP Fatal error:  Uncaught Error: Call to undefined function json_decode() in /var/www/scripts/tab_control.php:125
```

## Give the minimum Steps to Reproduce

1. Create container running `umple/umpleonline`
2. Select one of the examples in the WUI.
3. Observe the nginx access logs (`docker logs <container name>`)

## Describe the expected result that didn't happen

The example didn't load.

## Actual Result

 Just get the loading "spinning wheels" in the WUI.

### Workaround
Run `apk add php7-json` inside the container, and restart.

### Permanent fix
Add `php7-json` to `Dockerfile`

**Comment by TimLethbridge** [comment]
I have added php7-json to umpleonline/Dockerfile-base in PR #1455. I will The base is normally only created during a 'release'. I intend to do a release in the next week, that should clear this up.

**Comment by TimLethbridge** [comment]
This was fixed as of 1.30.1
